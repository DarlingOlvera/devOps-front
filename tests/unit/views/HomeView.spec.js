import { shallowMount } from "@vue/test-utils";
import Home from '@/views/HomeView.vue'

jest.mock('vue-router',()=>({
  useRouter: jest.fn(()=>({router:'login'}))
}))

describe('Pruebas en HomeView', () => {
  test('El html debe coincidir con el snaptshot', () => {
    const wrapper = shallowMount(Home)

   // expect(wrapper.html()).toMatchSnapshot()
  });
});
