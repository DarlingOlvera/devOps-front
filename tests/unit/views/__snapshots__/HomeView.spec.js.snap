// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HomeView El html debe coincidir con el snaptshot 1`] = `
<v-container class="d-flex justify-center align-center px-10" style="height: 100vh;">
  <div>
    <h2 class="text-center text-h4 text-title">Su sesión está iniciada</h2>
    <v-row class="d-flex justify-center my-10"><span class="text-secondary text-subtitle-1">Ahora puede acceder a la ruta que desee</span></v-row>
    <v-row class="d-flex justify-center">
      <v-btn color="primary" class="rounded-xl">Cerrar Sesión</v-btn>
    </v-row>
  </div>
</v-container>
`;
